operationsgateway_api/config.yml.example

```diff
@@ -12,13 +12,13 @@ images:
   # Colour maps for Matplotlib can be used
   default_colour_map: viridis
   colourbar_height_pixels: 16
-  echo_url: https://s3.echo.stfc.ac.uk
-  echo_username: username
-  echo_access_key: access_key
-  echo_secret_key: secret_key
-  image_bucket_name: test-bucket
-  upload_image_threads: 4
```

Comment:
upload_image_threads should still exist in the example config

Reply:
Fixed in 861ebc2e4d86e1d6bd48ef1eba48cb77c4a98c7d